randovania/games/fusion/gui/ui_files/preset_fusion_patches.ui

```diff
@@ -81,13 +81,28 @@
           <property name="title">
            <string>Room Design</string>
           </property>
-          <layout class="QVBoxLayout" name="unlock_layout"/>
+          <layout class="QVBoxLayout" name="unlock_layout">
+           <item>
+            <widget class="QCheckBox" name="anti_softlock_check">
+             <property name="text">
+              <string>Enable Anti-Softlock Room Edits</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QLabel" name="anti_softlock_label">
+             <property name="text">
+              <string>Enabling this will modify certain rooms to prevent potential softlocks for the player.</string>
```

Comment:
The plan for this was to have some logical consequences too, since some of antys patches remove the ability to do certain tricks.